8_build_project.py

```diff
@@ -1,252 +0,0 @@
-# Run this file to auto deploy the model, run a job, and deploy the application
```

Comment:
When I was making updates to some of Jeff's other AMP repo's, I intentionally included these (and even built out a new one) because Jeff advised not to break demo functionality for field team's whose customers do not have access to the Prototype Catalog... Are we sure we want to delete these?

Whatever the decision, I'll be sure to do the same across his other AMP repo's as well..